internals/scripts/gitlint.js

```diff
@@ -0,0 +1,15 @@
+// @ts-check
+const execSync = require('child_process').execSync
+
+const GIT_ORIGIN_REMOTE = 'origin'
+const RELEASE_BRANCH = 'master'
+const BRANCH = `${GIT_ORIGIN_REMOTE}/${RELEASE_BRANCH}`
+const COMMIT_SHA = require('child_process').execSync(`git rev-parse ${BRANCH}`).toString().trim()
```

Comment:
This looks like a rewrite of https://github.com/oasisprotocol/oasis-core/blob/50d972df71fed2bcaa88e6ce5430d919ec08087d/common.mk#L171-L180

Add a link in a comment